sdk/core/ci.yml

```diff
@@ -15,11 +23,30 @@ pr:
     include:
       - master
       - feature/*
+      - release/*
+      - hotfix/*
   paths:
     include:
       - sdk/core/
 
-jobs:
-  - template: ../../eng/pipelines/templates/jobs/archetype-sdk-client.yml
+stages:
+  - template: ../../eng/pipelines/templates/stages/archetype-sdk-client.yml
     parameters:
       ServiceDirectory: "*"
+      Artifacts:
+        - name: azure-abort-controller
+          safeName: azureabortcontroller
```

Comment:
I'm sure you have probably played with this but is there no way to remove this redundancy an generate these safenames?

Reply:
I haven't found one yet. I want to get rid of it as well - I did talk to the Pipelines team about adding a function to the language that can produced safe names and hashed values from strings as part of template processing which would solve this problem. But no commitment or ETA yet.